### ISSUE TITLE
ignore contigous bullets

### ISSUE THREAD

**Issue by nathanbraun** [lervag/vim-rainbow-lists#4]
This is really minor, but ideally it'd probably ignore bullets all in a row, e.g.

```
---
***
```

I know markdown for example treats them as horizontal lines.

**Comment by lervag** [comment]
Ah, yes. Do you have an example where this breaks something? I plan to add an example file to the repo based on the work in #1, I could also add an example for this particular thing.

Having a good example file makes it easier to develop, because I can quickly see if things are right or wrong.

**Comment by nathanbraun** [comment]
Sure, I noticed it with vim-zettel, which puts a title/date at the top of each file like this. For me the top line is multi colored (the bottom one isn't, so vim-zettel maybe is changing the coloring? not sure).

```
---
title: vim-rainbow-lists test file
date: 2021-02-25 10:35
---
```

**Comment by lervag** [comment]
Thanks, I've added this to the example file now and will fix it with #1.
